common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleService.java

```diff
@@ -199,11 +204,15 @@ protected void serviceInit(Configuration externalConf) throws Exception {
     _conf = new Configuration(externalConf);
     URL confOverlayUrl = Thread.currentThread().getContextClassLoader()
         .getResource(SHUFFLE_SERVICE_CONF_OVERLAY_RESOURCE_NAME);
+    String className = YarnShuffleService.class.getName();
     if (confOverlayUrl != null) {
-      logger.info("Initializing Spark YARN shuffle service with configuration overlay from {}",
-          confOverlayUrl);
       _conf.addResource(confOverlayUrl);
     }
+    String logsNamespace = _conf.get(SPARK_SHUFFLE_SERVICE_LOGS_NAMESPACE_KEY, "");
+    String loggerName = logsNamespace.isEmpty() ? className : className + " " + logsNamespace;
+    logger = LoggerFactory.getLogger(loggerName);
+    logger.info("Initialized Spark YARN shuffle service with configuration overlay from {}",
```

Comment:
We need a different message if `confOverlayUrl` is null, right? Maybe
```
if (confOverlayUrl == null) {
  logger.info("Initialized Spark YARN shuffle service using base configuration");
} else {
  logger.info(...); <- current logic here
}
```

Reply:
Added a default message, thanks